man/systemd-stub.xml

```diff
@@ -106,10 +115,13 @@
     DeviceTree in the corresponding EFI configuration table. systemd-stub will ask the firmware via the
     <literal>EFI_DT_FIXUP_PROTOCOL</literal> for hardware specific fixups to the DeviceTree.</para>
 
-    <para>The contents of eight of these nine sections are measured into TPM PCR 11. It is otherwise not used
-    and thus the result can be pre-calculated without too much effort. The <literal>.pcrsig</literal> section
-    is not included in this PCR measurement, since it is supposed to contain signatures for the output of the
-    measurement operation, and thus cannot also be input to it.</para>
+    <para>The contents of 11 of these 12 sections are measured into TPM PCR 11. It is otherwise not used and
+    thus the result can be pre-calculated without too much effort. The <literal>.pcrsig</literal> section is
+    not included in this PCR measurement, since it is supposed to contain signatures for the output of the
+    measurement operation, and thus cannot also be input to it. If an UKI contains multiple profiles, only
+    the PE sections of the selected profile are measured.</para>
```

Comment:
"only the PE sections of the selected profile are measured."

This sounds wrong, the base sections that are not overridden are also measured.